packages/immutable-rootfs/30cos-immutable-rootfs/cos-mount-layout.sh

```diff
@@ -216,12 +216,25 @@ function mountPersistent {
     echo "${mount#*:} ${mount%%:*} auto defaults 0 0\n"
 }
 
+function waitForPath {
+    local path=$1
+    local count=0
+
+    while [ ! -e "${path}" ]; do
+        sleep 1
+        udevadm settle
+        (( count++ ))
+        [ "${count}" -ge "${timeout}" ] && return 1
```

Comment:
nit: maybe we can move the `timeout` var declaration inside the function: it is just used here.